src/analytics/src/main/scala/osmesa/analytics/oneoffs/ChangesetStatsCreator.scala

```diff
@@ -37,12 +48,20 @@ object ChangesetStatsCreator
             )
             .orElse(Opts.env[URI]("DATABASE_URL", help = "The URL of the database"))
 
-        (historyOpt, changesetsOpt, databaseUrlOpt).mapN {
-          (historySource, changesetSource, databaseUrl) =>
+        (historyOpt, changesetsOpt, changesetBaseOpt, databaseUrlOpt).mapN {
+          (historySource, changesetSource, changesetBaseURI, databaseUrl) =>
             implicit val spark: SparkSession = Analytics.sparkSession("ChangesetStats")
             import spark.implicits._
 
+            val logger = org.apache.log4j.Logger.getLogger(getClass())
+
             val history = spark.read.orc(historySource)
+
+            val augdiffEndSequence = {
+              val t = history.select(max('timestamp)).first.getAs[java.sql.Timestamp](0).toInstant
+              ((t.getEpochSecond - 1347432900) / 60).toInt
+            }
```

Comment:
Move into `AugmentedDiffSource`?

Reply:
Is this relevant to OSM?  This may need to be removed, now that you mention it...

Reply:
It is. Overpass replication sequences are the same across all instances, regardless of source.

Reply:
Found in https://github.com/geotrellis/vectorpipe/pull/109/commits/d4c5d36430be79cb438dcc38954bb28adc9a7901